src/geckomat/change_model/applyCustomKcats.m

```diff
@@ -49,12 +49,17 @@
         error('Either send in a modelAdapter or set the default model adapter in the ModelAdapterManager.')
     end
 end
+
 params = modelAdapter.params;
 
 if nargin<2 || isempty(customKcats)
     customKcats = fullfile(params.path,'data','customKcats.tsv');
 end
 
+if ~exist(customKcats, 'file')
+   error(['Custom kcats cannot be applied because of missing expected file: ' customKcats]);
```

Comment:
Missing file should not throw an error, as this is not an essential function.

```suggestion
   disp(['No custom kcats are applied due to missing customKcats file: ' customKcats]);
   rxnUpdated = [];
   notMatch = [];
   return
```
A wrongly formatted file is more serious, so the other error on line 76 should stay.

Reply:
If I got this right, the whole point of `applyingCustomKcats` is to read these custom kcats from a file into the model structure. If there is no file, then there is no point of calling the function.

For example, if I create a file name with a typo, what should the pipeline do:
1. Warn me that no kcats have been applied, which assumes I carefully read the entire pipeline output.
2. Crash, thus making it obvious for me that this command is not doing what is expected of it.

Reply:
I was thinking of a pipeline, where the same list of functions are applied to multiple models. Sometimes the model has a `customKcats.tsv`, sometimes it doesn't. But I suppose in that pipeline you could first include a check if the file exists, before running the function.
Most often the functions are not run within such a pipeline though, and then indeed an error message would be warranted. I'll accept!